client/project_policy.go

```diff
@@ -18,6 +18,7 @@ type Policy struct {
 	ForceRemoteBackend          bool    `json:"forceRemoteBackend"`
 	DriftDetectionCron          string  `json:"driftDetectionCron"`
 	DriftDetectionEnabled       bool    `json:"driftDetectionEnabled"`
+	VcsPrCommentsEnabledDefault bool    `json:"vcsPrCommentsEnabledDefault"`
```

Comment:
can we put this one bellow `ContinuousDeploymentDefault`?
this is just my OCD yelling, and want those 3 to be together...